src/api/handlers/twilio/postReply.js

```diff
@@ -10,16 +10,17 @@ import {
 
 const convertReply = (reply) => {
   const languageConversions = {
-    yes: ["نعم", "አዎ", "haa", "Evet", "si", "yes"],
+    yes: ["نعم", "አዎ", "haa", "Evet", "si", "yes", "እወ"],
     // In spanish and english no is the same
-    no: ["لا", "አይ", "maya", "Hayır", "no"],
+    no: ["لا", "አይ", "maya", "Hayır", "no", "ኣይፋለይን"],
     interpreter: [
       "مترجم",
       "አስተርጓሚ",
       "turjubaan",
       "çevirmen",
       "interprete",
       "interpreter",
+      "ኣተርጓሚ"
```

Comment:
Is it just me or is there like 4 tabs of space on this line

Reply:
I am not sure what you mean but I have added the missing comma.